src/applications/personalization/preferences/PreferencesWidget.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+
+function AccordionWrapper({ children }) {
+  return (
+    <div className="usa-accordion">
+      <ul className="usa-unstyled-list">
+        {children}
+      </ul>
+    </div>
+  );
+}
+
+function AccordionItem({ onToggle, expanded, buttonText, name, children }) {
+  return (
+    <li>
+      <button className="usa-button-unstyled usa-accordion-button" aria-controls={name} aria-expanded={!!expanded} onClick={onToggle} name={name}>{buttonText}</button>
+      <div id={name} className="usa-accordion-content" aria-hidden={!expanded}>
+        <div itemProp="text">
+          {children}
+        </div>
+      </div>
+    </li>
+  );
+}
+
+class PreferencesWidget extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  handleAccordionToggle = (e) => {
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: !this.state[e.target.name],
+    });
+  }
+
+  render() {
+    // do not show in production
+    if (document.location.hostname === 'www.vets.gov') {
```

Comment:
Update to reflect post WBC `va.gov` state

Reply:
This now uses the `isProduction` helper.